src/TwitterAds/TailoredAudience/TailoredAudience.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace Hborras\TwitterAdsSDK\TwitterAds\TailoredAudience;
+
+use Hborras\TwitterAdsSDK\TwitterAds;
+use Hborras\TwitterAdsSDK\TwitterAds\Account;
+use Hborras\TwitterAdsSDK\TwitterAds\Resource;
+use Hborras\TwitterAdsSDK\TwitterAds\TailoredAudience\Exception\InvalidType;
+
+final class TailoredAudience extends Resource
+{
+    const RESOURCE_COLLECTION = 'accounts/{account_id}/tailored_audiences';
+    const RESOURCE = 'accounts/{account_id}/tailored_audiences/{id}';
+
+    const LIST_TYPE_EMAIL = 'EMAIL';
+    const LIST_TYPE_DEVICE_ID = 'DEVICE_ID';
+    const LIST_TYPE_TWITTER_ID = 'TWITTER_ID';
+    const LIST_TYPE_HANDLE = 'HANDLE';
+    const LIST_TYPE_PHONE_NUMBER = 'PHONE_NUMBER';
+
+    /** Writable */
+    protected $list_type;
+    protected $name;
+
+    protected $properties = [
+        'name',
+        'list_type',
+    ];
+
+    /** Read Only */
+    protected $deleted;
+    protected $targetable;
+    protected $audience_size;
+    protected $id;
+    protected $updated_at;
+    protected $created_at;
+    protected $audience_type;
+    protected $reasons_not_targetable;
+    protected $targetable_types;
+    protected $partner_source;
+
+    public function __construct(Account $account=null, $id=null)
+    {
+        parent::__construct($account);
+        $this->id = $id;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function delete()
```

Comment:
Yes, I agree that this is a bug @hborras 
To be honest, I was not sure why that was a put request in the first place :)